containers/ddev-webserver/ddev-webserver-base-scripts/start.sh

```diff
@@ -88,7 +88,7 @@ if [ ! -f ${NVM_DIR:-${HOME}/.nvm}/nvm.sh ]; then (install_nvm.sh || true); fi
 # should all be set up with both global and local
 # either way.
 if [ -d /mnt/ddev_config/.homeadditions ]; then
-    cp -r /mnt/ddev_config/.homeadditions/. ~/
+    cp -rL /mnt/ddev_config/.homeadditions/. ~/
```

Comment:
Unfortunately this approach can't work as it is. The fact is that the symlink (/mnt/ddev_config/.homeadditions/.ssh/config, for example) is already disconnected from the source, because it's mounted into the container where the symlink can't be followed. For this to work, the copy has to be done on the host side, not by following the symlink here.